modules/virt-querying-metrics.adoc

```diff
@@ -31,6 +31,17 @@ A value above '0' means that the vCPU wants to run, but the host scheduler canno
 ====
 To query the vCPU metric, the `schedstats=enable` kernel argument must first be applied to the `MachineConfig` object. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler.
 ====
+kubevirt_vmi_vcpu_delay_seconds_total::
+Returns the cumulative time, in seconds, that a vCPU was enqueued by the host scheduler but could not run immediately.  
+This delay appears to the virtual machine as steal time and can affect performance, often indicating CPU overcommitment or contention on the host.  
```

Comment:
Suggestion for future improvement: define "steal time" in case the reader doesn't know what it means.